api/src/main/java/uk/gov/dhsc/htbhf/hmrc/entity/Adult.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.dhsc.htbhf.hmrc.entity;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import java.time.LocalDate;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.Size;
+
+@Entity
+@Data
+@SuperBuilder
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@Table(name = "hmrc_ctc_adult")
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
+@SuppressWarnings("PMD.TooManyFields")
+public class Adult extends BaseEntity {
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "hmrc_ctc_household_id", nullable = false)
+    private Household household;
+
+    @Size(max = 9)
+    @Column(name = "nino")
+    private String nino;
+
+    @Column(name = "award_date")
+    private LocalDate awardDate;
+
+    @Size(max = 35)
+    @Column(name = "surname")
+    private String surname;
+
+    @Size(max = 35)
+    @Column(name = "first_forename")
+    private String firstForeName;
```

Comment:
Forename in this field and the one below should not have a capital N for name.

Reply:
Updated.